ja/doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -318,15 +312,14 @@ def wait_for_state_update(self, box_is_known=False, box_is_attached=False, timeo
 
     ## BEGIN_SUB_TUTORIAL wait_for_scene_update
     ##
-    ## Ensuring Collision Updates Are Received
+    ## 衝突の更新がされているか確かにする
     ## ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The compiler returns following Warning.
```log
WARNING: Title underline too short.
```
So, please fix this part.

Reply:
I fixed this issue on 
c9d78a7 .
so please check it

Reply:
I checked this is fixed !
Thank you.